blosc/frame.c

```diff
@@ -671,13 +673,15 @@ void* blosc2_frame_append_chunk(blosc2_frame* frame, void* chunk) {
   int64_t frame_len;
   int64_t nbytes;
   int64_t cbytes;
-  uint32_t chunksize;
+  uint32_t ichunksize;
```

Comment:
Maybe `uchunksize` should be a better name for unsigned ints?

Reply:
sure, renaming would make sense. I'm sure there is a reason, but ideally the chunksize would be the same datatype everywhere ;)